### DISCUSSION TITLE
Some blacklists aren't explicit

### DISCUSSION THREAD

**Discussion by amano-kenji** [netblue30/firejail#5792] [General]
For example,

https://github.com/netblue30/firejail/blob/490051f71494e828f2020257c012c3b28398d852/src/firejail/fs.c#L744

I had to figure this out through trials and errors.

**Comment by rusty-snake** [comment]
What is this?

A change request? A bug report? An information? A request for better Ddcumentation?

If you drop it in a discussion without telling more it is not really helpful.

**Reply by amano-kenji** [reply]
I wish it was easy to figure out that /sys/module and other directories were blacklisted. For now, I have to dig through the codebase.

**Reply by rusty-snake** [reply]
There is `--debug`.

**Reply by amano-kenji** [reply]
I think `--debug-blacklists` would be less verbose. I think it would be helpful for the man page to mention that there are hidden blacklists that you should uncover with `--debug-xxx`.

My assumption was that all blacklists were in /etc/firejail or ~/.config/firejail.

**Reply by jjakob** [reply]
I think better documentation is needed. @rusty-snake helped me understand it much better:  https://github.com/netblue30/firejail/discussions/5791#discussioncomment-5634243
For a start, the manpage should be expanded with this information. The description of `--blacklist` does not explain it well enough. Examples like in that comment would also be great.
Probably the same for `--whitelist`.

**Reply by rusty-snake** [reply]
btw, contributions (including documentation) are always welcome. As a lot other FLOSS project lives firejail from the work of all its contributors done in their free time.

**Reply by rusty-snake** [reply]
Actually there is some documentation about it (beside reading the code) in [noprofile.profile](https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/noprofile.profile).

**Reply by amano-kenji** [reply]
@jjakob I think you are in the best position to add some documentation to the man page.
